jobManager.py

```diff
@@ -82,6 +82,9 @@ def __manage(self):
         elif Config.VMMS_NAME == "ec2SSH":
             from vmms.ec2SSH import Ec2SSH
             vmms = Ec2SSH()
+        elif Config.VMMS_NAME == "dockerSSH":
+            from vmms.dockerSSH import DockerSSH
+            vmms = DockerSSH()
 
```

Comment:
Hmm, ok. Yeah exiting on a higher level makes more sense. But we have to make sure that it exits because of a problem that affects all of Tango and not just because of one job. It shouldn't be too big of a problem because worker threads have their own exception handler and so any stray job won't bring the whole system down.